dev/com.ibm.ws.microprofile.config.1.4/src/com/ibm/ws/microprofile/config14/impl/Config14Impl.java

```diff
@@ -47,13 +50,14 @@
      */
     public Config14Impl(ConversionManager conversionManager, SortedSources sources, ScheduledExecutorService executor, long refreshInterval) {
         super(conversionManager, sources);
+        rawValueCache = new TimedCache<>(executor, 500, TimeUnit.MILLISECONDS);
```

Comment:
The 500 should be in a constant somewhere, possibly a constants file and possibly configurable??

Reply:
Unless you object, I'll do that as a follow up.